src/Service/Organization/PackageManager.php

```diff
@@ -67,9 +67,11 @@ public function saveProvider(array $json, string $organizationAlias, string $pac
         $filepath = $this->filepath($organizationAlias, $packageName);
 
         $dir = \dirname($filepath);
-        $this->repoFilesystem->createDir($dir);
+        if (!$this->repoFilesystem->has($dir)) {
+            $this->repoFilesystem->createDir($dir);
+        }
 
-        $this->repoFilesystem->write($filepath, \serialize($json));
+        $this->repoFilesystem->put($filepath, \serialize($json));
```

Comment:
write will throw exception if file exist :wink:

Reply:
Now I got this, you actually provided the fix. Thx.